File: src/main/java/com/baudoin/sofascore/FootballController.java
package com.baudoin.sofascore;

import org.springframework.http.MediaType;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;
import org.springframework.web.bind.annotation.RestController;

@RequestMapping("/api")
@RestController
public class FootballController {

    @RequestMapping(method = RequestMethod.GET)
    public String index() {
        return "Test Ok";
    }
}

File: src/main/java/com/baudoin/sofascore/FootballController.kt
package com.baudoin.sofascore

import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RequestMethod
import org.springframework.web.bind.annotation.RestController

@RequestMapping("/api")
@RestController
class FootballController {

    @RequestMapping(method = [RequestMethod.GET])
    fun index(): String {
        HttpUtils.setNetworkManagerInterfaces()
        FootballNetworkManager.getEvents("2019-04-22", object: CallBackManagerWithError<FootballResponse>{
            override fun onSuccess(response: FootballResponse) {

            }

            override fun onError(pError: String) {

            }

        })
        return "ok"
    }
}